save-backend/src/main/kotlin/com/saveourtool/save/backend/utils/ApiGroupsConfiguration.kt

```diff
@@ -21,7 +21,10 @@ class ApiGroupsConfiguration {
     fun openApiAll(): GroupedOpenApi? = GroupedOpenApi.builder()
         .group("all")
         .pathsToMatch("/api/**")
-        .packagesToScan("com.saveourtool.save.backend.controllers")
+        .packagesToScan(
+            "com.saveourtool.save.backend.controllers",
```

Comment:
if you will have the time, could you please add to the readme the link to this file, I guess anyone will forget about this config, but now we start to actualize our readme's, probably it will be usefull too

Reply:
to any readme

Reply:
also, I didnt remember why I put it to the `utils` of backend , probably its better to put it to `config`

Reply:
updated